lldb/test/API/functionalities/process_save_core_minidump/TestProcessSaveCoreMinidump.py

```diff
@@ -132,8 +132,13 @@ def test_save_linux_mini_dump(self):
                 stacks_to_sp_map,
             )
 
+            options = lldb.SBCoreDumpOptions()
+            core_sb_stack_spec = lldb.SBFileSpec(core_sb_stack)
+            options.SetOutputFile(core_sb_stack_spec)
+            options.SetCoreDumpPluginName("minidump")
```

Comment:
Check error here and try setting it to an invalid value:
```
error = options.SetPluginName("bad")
# verify error is set
error = options.SetPluginName("minidump")
# verify error is in success state
```
Or you can test the SBCoreDumpOptions in a separate test by calling accessors with a variety of bad and good input.

Reply:
Created a new unit test file for this